client/elife-theme/src/elements/xpub-edit/index.js

```diff
@@ -0,0 +1,28 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+export const MenuBar = {
+  Wrapper: css`
+    border: ${th('borderWidth')} ${th('borderStyle')} ${th('colorBorder')};
+    border-radius: ${th('borderRadius')} ${th('borderRadius')} 0 0;
+    background-color: ${th('colorBackgroundHue')};
+    margin-bottom: 0;
+    border-bottom: none;
+    line-height: calc(${th('gridUnit')} * 2);
+    padding: ${th('space.1')};
+  `,
+}
+
+export const MenuButton = {
+  Button: css`
+    background-color: transparent;
+    margin: ${th('space.1')}
+    width: ${th('space.3')};
+    /* subtract border width */
+    height: calc(${th('space.3')} - 2px);
```

Comment:
2px hardcoded?

Reply:
Yeah the 2px border is part of the core component. I'm sure it won't stay this way but the designs don't specify any visual style for buttons that are active.